Travis Builds Fail on 3.5

Our builds keep failing on 3.5 for some reason, but succeeding on 3.6. Seems to be erroring on calls to `main.py init` in the `test_cli.py`. Annoyingly, all of these tests pass on my box running 3.5, so it might just be a Travis environment problem? Considering we have been thinking about breaking apart `main.py` anyway, it might be worth disabling these tests for now.

Here's a dump of the errors (from [here](https://travis-ci.org/Halibot/halibot/jobs/501794221) ):
```
0.53s$ ./test.sh
.......
----------------------------------------------------------------------
Ran 7 tests in 0.007s
OK
FFF
======================================================================
FAIL: test_fetch (__main__.TestFetch)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/test_cli.py", line 46, in test_fetch
    self.cmd(['init'])
  File "tests/test_cli.py", line 20, in cmd
    self.assertEqual(proc.returncode, 0)
AssertionError: 1 != 0
======================================================================
FAIL: test_init (__main__.TestInit)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/test_cli.py", line 29, in test_init
    self.cmd(['init'])
  File "tests/test_cli.py", line 20, in cmd
    self.assertEqual(proc.returncode, 0)
AssertionError: 1 != 0
======================================================================
FAIL: test_install (__main__.TestInstall)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/test_cli.py", line 58, in test_install
    self.cmd(['init'])
  File "tests/test_cli.py", line 20, in cmd
    self.assertEqual(proc.returncode, 0)
AssertionError: 1 != 0
----------------------------------------------------------------------
Ran 3 tests in 0.060s
FAILED (failures=3)
```

--- a/main.py
+++ b/main.py
@@ -209,23 +209,23 @@ def h_info(args):
 			print("No such agent or module")
 			return
 
-		print(f"\n{args.object_name}: ({conf['of']})")
+		print("\n{args.object_name}: ({conf['of']})".format(args=args, conf=conf))
 		for k in conf:
 			if k != "of":
-				print(f"  {k}: {conf[k]}")
+				print("  {k}: {conf[k]}".format(k=k, conf=conf))
 	else:
 		# Show all configured objects
 		if len(bot.config.get("agent-instances", {})) > 0:
 			print("\nConfigured agents:")
-			agents = bot.config["agent-instances"]
+			agents = bot.config.get("agent-instances")
 			for name in agents:
-				print(f"  {name} ({agents[name]['of']})")
+				print("  {name} ({agents[name]['of']})".format(name=name, agents=agents))
 
 		if len(bot.config.get("module-instances", {})) > 0:
 			print("\nConfigured modules:")
 			modules = bot.config.get("module-instances")
 			for name in modules:
-				print(f"  {name} ({modules[name]['of']})")
+				print("  {name} ({modules[name]['of']})".format(name=name, modules=modules))
 
 	print("")
 

